### PR TITLE
#28 adds styling to appointments page

### PR DESCRIPTION
**Pull request by Jen-Harris**
allaboutme-appointments screen

### CODE REVIEW

**Comment by Jen-Harris** [comment]
queries need updating for people logging in for the first time

File: public/theme.css [hunk 1]
```diff
@@ -36,6 +46,7 @@
     line-height: 30px; /* To center it vertically */
     color: white;
```
Comment (rebecacalvoquintero) [inline public/theme.css]: missing brace

File: src/app.js [hunk 2]
```diff
@@ -31,16 +31,16 @@ app.use(cookieSession({
...
-app.use('/info_page', requireLogin);
-app.use('/about', requireLogin);
-app.use('/symptoms', requireLogin);
-app.use('/appointments', requireLogin);
-app.use('/background', requireLogin);
-app.use('/colour_scheme', requireLogin);
-app.use('/send', requireLogin);
-app.use('/progress', requireLogin);
-app.use('/finish', requireLogin);
+// app.use('/home', requireLogin);
```
Comment (dangerdak) [inline src/app.js]: These need uncommenting :)

**Comment by Jen-Harris** [comment]
@dangerdak @polyccon @rebecacalvoquintero I'm not sure why this is failing. Any ideas?

File: src/app.js (hunk 2 above)
Comment (Jen-Harris) [reply src/app.js]: This is done now 👍

**Pull request merged**